CDC.WebNotes.Api/Controllers/NotesController.cs

```diff
@@ -103,5 +107,17 @@ public async Task<IActionResult> DeleteNote([FromRoute] int id)
                                    "Notes",
                                    new { noteId = id });
         }
+
+        [HttpPost("{id}/attachments")]
+        public async Task<IActionResult> PostAttachment([FromRoute] int id,
+                                                           [FromBody] CreateAttachment createAttachment)
+        {
+
+            AttachmentDto createAttachmentDto = _mapper.Map<AttachmentDto>(createAttachment);
+
+            await _notesService.AddAttachment(id, createAttachmentDto);
+
+            return StatusCode(201);
```

Comment:
```suggestion
            return CreatedAtAction(nameof(AttachmentsController.Get),
                                   "Attachments",
                                   new { noteId = attachment.Id });
```
Pls return attachment in service.Create() method